content/events/jobcrypt-web3-&-blockchain-talks-munich-germany.md

```diff
@@ -0,0 +1,9 @@
+---
+eventName: JobCrypt Web3 & Blockchain Talks Munich Germany, Sponsored by the Filecoin Orbit Programme
+location: "Munich, Germany"
+website: [https://www.jobcrypt.com/pages/promotions/munich/2023-01-25-In-Person-Talk.html](https://www.jobcrypt.com/pages/promotions/munich/2023-01-25-In-Person-Talk.html)
```

Comment:
@cryptotwilight can you please change this from markdown to just URL "https://..."

Thanks!